src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -187,6 +187,11 @@
                                                     <div class="tab">
                                                         <h3 class="tab-title">Attachment #{{ loop.index }}</h3>
                                                         <div class="tab-content">
+                                                            <p>
+                                                                <a href="data:{{ attachment.contentType|default('application/octet-stream') }};base64,{{ collector.base64Encode(attachment.body) }}"{% if attachment.filename %} download="{{ attachment.filename }}"{% endif %}>
```

Comment:
```suggestion
                                                                <a href="data:{{ attachment.contentType|default('application/octet-stream') }};base64,{{ collector.base64Encode(attachment.body) }}" download="{{ attachment.filename|default('attachment') }}">
```

I tested creating a `data:` anchor tag without the `download` attribute and nothing happens when I click it (in Chrome). When I set it to a filename without an extension it does download the file. For a content-type of `application/octet-stream` it would download as `attachment` (no extension) but if `application/pdf` it downloads as `attachment.pdf`. I guess Chrome, at least, auto-adds the extension for known content-types?

Can you confirm this on your end?